Remotion/ObjectBinding/Web/UI/Controls/BocCompoundColumnDefinition.cs

```diff
@@ -32,51 +33,30 @@ namespace Remotion.ObjectBinding.Web.UI.Controls
   public class BocCompoundColumnDefinition : BocValueColumnDefinition
```

Comment:
Please make sure you merge this PR after [RM-8828](https://re-motion.atlassian.net/browse/RM-8828) is done so we only have the actual semantic changes.

Reply:
Will undo these changes, [RM-8828](https://re-motion.atlassian.net/browse/RM-8828) is supposed to get merged after RelationsToInterfaces, because we should not only cleanup this part of re-motion. 

Thought I actually already undid the changes, but somehow shoving git branches around every 2 seconds and having the state in 2 different places is...hard?